Cesium3DTilesContent/src/BatchTableToGltfStructuralMetadata.cpp

```diff
@@ -2073,38 +2073,45 @@ struct BatchIdSemantic {
       return;
     }
     uint32_t byteOffset = byteOffsetIt->value.GetUint();
+    std::optional<MetadataProperty::ComponentType> componentType;
```

Comment:
It doesn't look like `componentType` would ever be nullopt. if `componentTypeIt` doesn't exist, then it defaults. If it exists, but is invalid, then the function returns. It'd be cleaner if this wasn't an optional, and instead started off with the default value. Then, it can be overridden as necessary.

```suggestion
    MetadataProperty::ComponentType componentType = MetadataProperty::ComponentType::UNSIGNED_SHORT;
```